src/ol_openedx_chat_xblock/ol_openedx_chat_xblock/block.py

```diff
@@ -150,6 +151,21 @@ def ol_chat(self, request, suffix=""):  # noqa: ARG002, PLR0911
         try:
             block_id = self.usage_key.block_id
 
+            # Common tracker data
+            tracker_base_data = {
+                "block_id": block_id,
+                "canvas_course_id": self.course_id,
```

Comment:
Is this plugin intended to be used only for Canvas course? if not, could we use course_id? The open edx tracking logs are following [this convention](https://docs.openedx.org/en/latest/developers/references/internal_data_formats/tracking_logs/common_fields.html#context-member-fields-common-to-all-events) for common fields we've added to our open edx tracking log tables in the data platform. It would be great we could use the same field name as other open edx events for consistency. Also, if its easy, could we capture `org_id` as well?  something like 'MITx`